cypress/objects/pageComponents.js

```diff
@@ -0,0 +1,18 @@
+class PageComponents {
+    getUrl(){
+        return cy.url()
+    }
```

Comment:
this function is kind of redundant, it does not take any parameters nor do anything apart from calling another function. If it would be in any of the page objects you cold do this instead: 

validateUrl() {
      cy.url().should('include', '/time-logging')
}